analyzers/its/expected/Nancy/Nancy--net452-S2259.json

```diff
@@ -0,0 +1,17 @@
+{
+"issues":  [
+{
+"id":  "S2259",
+"message":  "'viewName' is null on at least one execution path.",
+"location":  {
+"uri":  "sources\Nancy\src\Nancy\ViewEngines\DefaultViewFactory.cs",
+"region":  {
+"startLine":  58,
+"startColumn":  34,
+"endLine":  58,
+"endColumn":  42
+}
+}
+}
+]
+}
```

Comment:
This is a false positive. `viewName` can only be null if `model` is not null, in which case `viewName.Length` is not accessed.
https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/sources/Nancy/src/Nancy/ViewEngines/DefaultViewFactory.cs#L51